src/Components/PublishablePositions/PublishablePositions.jsx

```
@@ -193,6 +196,19 @@ const PublishablePositions = ({ viewType }) => {
     selectedBidCycles,
   ]);
 
+  const exportPublishablePositions = () => {
+    if (!exportIsLoading) {
+      setExportIsLoading(true);
+      publishablePositionsExport(getQuery())
+        .then(() => {
+          setExportIsLoading(false);
+        })
+        .catch(() => {
+          setExportIsLoading(false);
+        });
+    }
+  };
```

so - I'm a little confused on why the loading stuff here is in the local state and not in the action/reducer, like every other `isLoading` state 🤔

also - it doesn't even DO anything, like trigger a loading spinner or something 🤷

---

i could definitely put it into the action/reducer files, just saw that numerous other exports handled the loading state as a local state so i figured it was fine to shorthand it like this. also uncommented out the isLoading prop of the exportbutton so now the loading state is in use. 

also i can handle the 'fails silently' comment on the API pr here - can fire a toast from here on failure